### PR TITLE
Fix130910 formatting of properties and ifs

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by Pilchie** [comment]
Note: the first two commits are from #5478, which I expect to merge first, because this change builds on them.

**Comment by Pilchie** [comment]
Tagging @dotnet/mlangide @heejaechang @basoundr for code review.

File: src/Workspaces/CSharp/Portable/Formatting/Rules/IndentBlockFormattingRule.cs [hunk 1]
```diff
@@ -223,11 +223,19 @@ private void AddEmbeddedStatementsIndentationOperation(List<IndentBlockOperation
...
             {
-                if (!(elseClause.Statement is BlockSyntax || elseClause.Statement is IfStatementSyntax))
+                if (elseClause.Statement is BlockSyntax)
                 {
-                    AddEmbeddedStatementsIndentationOperation(list, elseClause.Statement);
+                    return;
                 }
 
+                if (elseClause.Statement is IfStatementSyntax)
+                    //&& elseClause.ElseKeyword.Span == ((IfStatementSyntax)elseClause.Statement).IfKeyword.Span.Line)
```
Comment (Pilchie) [inline src/Workspaces/CSharp/Portable/Formatting/Rules/IndentBlockFormattingRule.cs]: Anyone know the right way to check this condition from inside a rule?

File: src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs [hunk 2]
```diff
@@ -960,25 +960,33 @@ public static SyntaxNode GetParent(this SyntaxNode node)
...
+                   node is IfStatementSyntax ||
+                   node is LabeledStatementSyntax ||
+                   node is LockStatementSyntax ||
                    node is UsingStatementSyntax ||
-                   node is DoStatementSyntax;
+                   node is WhileStatementSyntax;
         }
 
-        public static StatementSyntax GetEmbeddedStatement(this SyntaxNode node)
+        unsafe public static StatementSyntax GetEmbeddedStatement(this SyntaxNode node)
```
Comment (DustinCampbell) [inline src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs]: huh?

Comment (Pilchie) [inline src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs]: Oops.  Was experimenting in place. :blush:

**Comment by Pilchie** [comment]
Okay @heejaechang and @basoundr - care to take a look at this.  It seems to work now.

**Comment by davkean** [comment]
tag @dotnet/roslyn-ide.

@heejaechang and @basoundr can you look at this?

**Comment by Pilchie** [comment]
Actually, I'm closing this for now, as it doesn't work with projected spans in asp.net properly.

**Pull request closed without merging**